admin/api_test.go

```diff
@@ -252,3 +253,51 @@ func TestGetState(t *testing.T) {
 		}
 	}
 }
+
+func TestSetConnectors(t *testing.T) {
+	tests := []struct {
+		connectors []connector.ConnectorConfig
+	}{
```

Comment:
that's fine - failure is tested in the integration section so I think we have decent coverage.